### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (.mtk20:nth-child(1))

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2328]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[Microsoft MakeCode for micro:bit](https://makecode.microbit.org/beta#editor)
#### Element path
.mtk20:nth-child(1)
#### Snippet
`<span class="mtk20">basic</span>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.82 (foreground color: #1e90ff, background color: #ecf0f1, font size: 15.0pt, font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

When using functions from a namespace that has insufficient contrast, the text in the monaco editor will have insufficient contrast:

<img width="718" alt="Screen Shot 2019-08-02 at 2 19 55 PM" src="https://user-images.githubusercontent.com/5615930/62399418-a272c900-b530-11e9-9fa6-0afbcf6f26d1.png">

This issue does not reproduce / is handled properly in high contrast mode

**Comment by abchatra** [comment]
This is fixed in high contrast mode..
